angio-frontend/src/modules/user/components/AccountActivationForm.vue

```diff
@@ -0,0 +1,174 @@
+                                prepend-inner-icon="lock"
+                                counter
+                                maxlength="120"
+                                outline
+                                required
+                        ></v-text-field>
+                        <v-text-field
+                                v-model="newPasswordRepeat"
+                                v-bind:label="$t('user.component.activation.activationFormStep.newPasswordRepeat.field')"
+                                v-bind:rules="[v => !!v || $t('user.component.activation.activationFormStep.newPassword.validation.NotEmpty'),
+                                    v => v.length >= 8 || $t('user.component.activation.activationFormStep.newPassword.validation.MinLength'),
+                                    form.newPassword === newPasswordRepeat || $t('user.component.activation.activationFormStep.newPassword.validation.DoesNotMatch')]"
+                                type="password"
+                                name="password"
+                                prepend-inner-icon="lock"
+                                counter
+                                maxlength="120"
+                                outline
+                                required
+                        ></v-text-field>
+                        <v-expand-transition>
+                            <ul v-if="displayError" data-test-id="formMsg__list">
+                                <li class="red--text body-1">
+                                    {{ $t('user.component.activation.activationFormStep.error') }}
+                                </li>
+                            </ul>
+                        </v-expand-transition>
+                    </v-form>
+                </v-card-text>
+            </v-window-item>
+
+            <v-window-item v-bind:value="2">
+                <v-card-title class="headline font-weight-regular justify-space-between">
+                    <span>{{ $t('user.component.activation.activationSuccessStep.title') }}</span>
+                    <v-spacer></v-spacer>
+                </v-card-title>
+                <v-card-text class="subheading font-weight-light">
+                    <span>{{ $t('user.component.activation.activationSuccessStep.content') }}</span>
+                </v-card-text>
+            </v-window-item>
+        </v-window>
+
+        <v-divider></v-divider>
+
+        <v-card-actions>
+            <v-spacer></v-spacer>
+            <v-btn
+                    v-show="step === 1"
+                    v-bind:disabled="!valid"
+                    v-bind:loading="fetching"
+                    form="activation-form"
+                    type="submit"
+                    color="primary"
+                    flat
+                    round
+            >
+                {{ $t('user.component.activation.activationFormStep.button.send') }}
+            </v-btn>
+            <v-btn
```

Comment:
Move to center of second step

Reply:
Done

Reply:
Not to center of bottom of second step but to center bellow icon and text like "home" button on the 404 page.

Reply:
Done